lib/alephant/renderer/views/html.rb

```diff
@@ -1,64 +1,57 @@
-require "alephant/renderer/views/base"
-require "mustache"
-require "i18n"
+require 'alephant/renderer/views/base'
+require 'alephant/renderer/engine/mustache'
+require 'alephant/renderer/i18n/locale_component_yaml'
 
 module Alephant
   module Renderer
     module Views
-      class Html < Mustache
+      class Html
         include ::Alephant::Renderer::Views::Base
 
+        class << self
+          # FIXME: remove this when we've gotten rid of all of the `self.template_path` directives
```

Comment:
Will be a big task with all the old renderers at the moment, many election ones, Newsbeat, AMP and many others to check.